src/roll20/content-script.js

```diff
@@ -741,6 +745,14 @@ async function handleOGLRenderedRoll(request) {
         atk_props["dmg2"] = convertRollToText(request.whisper, request.damage_rolls[1][1]);
         atk_props["dmg2type"] = request.damage_rolls[1][0];
     }
+    // If we have a character-related roll, then add modfier text based on the request and character settings
+    if (originalRequest.character && originalRequest.character.type === "Character") {
+        atk_props["mod"] = (originalRequest["to-hit"] || "").trim();
+        const custom_roll_dice = (originalRequest.character.settings["custom-roll-dice"] || "").trim();
```

Comment:
actually... the `mod` should be set regardless of the character type, it's just the `custom-roll-dice` that  that should only be added for characters.